drop-ins/object-cache.php

```diff
@@ -150,9 +152,28 @@ function wp_cache_incr( $key, $offset = 1, $group = '' ) {
 function wp_cache_init() {
     global $wp_object_cache;
 
+    if ( ! defined( 'WP_REDIS_PREFIX' ) && getenv( 'SPINUPWP_CACHE_KEY_SALT' ) ) {
+        define( 'WP_REDIS_PREFIX', getenv( 'SPINUPWP_CACHE_KEY_SALT' ) );
+    }
+
+    if ( ! defined( 'WP_REDIS_SELECTIVE_FLUSH' ) && getenv( 'WP_REDIS_SELECTIVE_FLUSH' ) ) {
+        define( 'WP_REDIS_SELECTIVE_FLUSH', (bool) getenv( 'WP_REDIS_SELECTIVE_FLUSH' ) );
+    }
```

Comment:
This should always be `true` in Spinup https://github.com/deliciousbrains/spinupwp-plugin/commit/7cc2b8c68cb3ff886431bdfe8511657f6503ac67#diff-e176a90a3c73dc81c0254b1712e3a7e65b13375641c60a4e8836af1bbe8958ad